obspy/imaging/waveform.py

```diff
@@ -37,6 +40,10 @@
 from obspy.core.util import (FlinnEngdahl, createEmptyDataChunk,
                              locations2degrees)
 from obspy.core.util.decorator import deprecated_keywords
+from obspy.imaging.util import ObsPyAutoDateFormatter
+
+
+MINMAX_ZOOMLEVEL_WARNING_TEXT = "Warning: Zooming into MinMax Plot!"
```

Comment:
Well, you're right supposing the user explicitely requested a fast/minmax plot.

But.. the fast/minmax option is used automatically for high npts and I am fairly sure that most people are not aware of this fact. I assume most people in that case would recognize the fact that the strange looks when zooming in are artifacts from the plotting, but if only one person would be held up for ten minutes wondering about the seismogram I think the warning message is already justified..